### PR TITLE
Translations for auth

### PR DESCRIPTION
**Pull request by ashwinraghav**
- This change imports translations for firebase UI auth. Issues related to formatting (particularly for RTL languages) are to be expected. Fixes are welcome 👍 
- Once we see some stability with the UI for translations, we could roll forward with publishing version 11.0.1.

### CODE REVIEW

**Comment by samtstern** [comment]
I'm gonna write some scripts to clean up these translations.

**Comment by samtstern** [comment]
TODO:
screenshot from 2017-06-22 08 57 47

**Comment by samtstern** [comment]
The only thing we're missing now is a translation for `privacy_policy`.

**Comment by samtstern** [comment]
@ashwinraghav did you have a reason for disabling the `MissingTranslation` lint?  I think we need that to make sure we have complete coverage.

File: auth/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <string name="verified" translation_description="Progress dialog success transition when phone number was verified">Geverifieerd</string>
+    <string name="code_sent" translation_description="Progress dialog success transition once sms is sent">Code verzonden</string>
+    <string name="resending" translation_description="Progress dialog text when user clicks on resend sms">Opnieuw verzenden…</string>
+    <string name="resend_code" translation_description="Link text to resend verification sms">Code opnieuw verzenden</string>
+    <string name="verify_phone_number" translation_description="Button text to submit confirmation code and complete phone verification">Telefoonnummer verifiëren</string>
+    <string name="continue_phone_login" translation_description="Button text to submit phone number and send sms">Doorgaan</string>
+    <string name="sms_terms_of_service" translation_description="Fine print warning displayed below Verify Phone Number button">Als u op \'%1$s\', ontvangt u mogelijk een sms. Er kunnen sms- en datakosten in rekening worden gebracht.</string>
```
Comment (wwolters) [inline auth/src/main/res/values-nl/strings.xml]: A verb is missing in this sentence. It should be `Als u op \'%1$s\' tikt, ontvangt u mogelijk een sms. Er kunnen sms- en datakosten in rekening worden gebracht.`

Comment (samtstern) [inline auth/src/main/res/values-nl/strings.xml]: Thanks for this!

**Comment by samtstern** [comment]
For some reason when I run the app in a non-english locale I see "ui_flow" in the title bar.  Need to investigate.

**Pull request merged**